contracts/GenArt721CoreV3_Engine.sol

```diff
@@ -408,6 +410,12 @@ contract GenArt721CoreV3_Engine is
         // initialize next project ID
         _nextProjectId = _startingProjectId;
         emit PlatformUpdated(FIELD_NEXT_PROJECT_ID);
+        // register contract as an Engine contract
+        IEngineRegistryV0(_engineRegistryContract).registerContract(
```

Comment:
I think no one would fork the Engine contract to use with their own infra (would imagine they would just use the normal core contract) but if we do imagine that might be a possibility should we handle the case that the registry contract doesn't exist?

Reply:
@yoshiwarab in that case couldn't they just deploy a variant that removes this line and constructor argument? I think that addresses what you're pointing out, but is preferable to making the constructor argument here optional. WDYT ser?